camel-core/src/main/java/org/apache/camel/model/ThrottleDefinition.java

```diff
@@ -55,14 +60,26 @@
     private Boolean callerRunsWhenRejected;
     @XmlAttribute
     private Boolean rejectExecution;
-    
+
     public ThrottleDefinition() {
     }
 
     public ThrottleDefinition(Expression maximumRequestsPerPeriod) {
         super(maximumRequestsPerPeriod);
     }
 
+    public ThrottleDefinition(Expression correlationExpression, Expression maximumRequestsPerPeriod) {
```

Comment:
These 2 constructors are using different order of the parameters, eg it would be good to keep it consistent. As correlation expression is a new parameter, I would add it as 2nd.